### PR TITLE
Storybook

### PR DESCRIPTION
**Pull request by GiantRobots**
…apper to come

### How was this tested 👻 

- [ ] Manual testing
- [ ] E2E tests added
- [ ] Unit tests added

### CODE REVIEW

File: src/stories/Button.stories.ts
Comment (rossedfort) [inline src/stories/Button.stories.ts:1]: I'm wondering if we want to throw stories in holocene, i.e. `src/lib/holocene/stories` ?

File: src/stories/ReadMe.mdx
Comment (rossedfort) [inline src/stories/ReadMe.mdx:1]: What is this file, and do we need/want it? If so - can we expand on its contents?

Comment (GiantRobots) [reply src/stories/ReadMe.mdx:1]: It's a top level `README` the plan is definitely to fill it out as I get more information on what good ideas are

File: src/stories/Button.stories.ts
Comment (GiantRobots) [reply src/stories/Button.stories.ts:1]: I think that's a good next step.

 I'm unfamiliar with how storybook is configured these days and figure we can do this temporarily and then move it to holocene/stories. Also probably good to keep them split for a time until we do a proper effort to migrate (and verify the utility of this)

File: src/lib/holocene/stories/Button.stories.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  component: ButtonWrapper,
+  argTypes: {
+    leadingIcon: { ...iconControls },
+    trailingIcon: { ...iconControls },
+    variant: {
+      control: {
+        type: 'select',
+      },
+      options: ['primary', 'secondary', 'destructive', 'ghost'],
+    },
```
Comment (rossedfort) [inline src/lib/holocene/stories/Button.stories.ts:26]: Maybe a future project, try to type this such that we can only have controls with valid prop names/values.

**Review by rossedfort — APPROVED** [review]
🎨

File: src/lib/holocene/stories/Button.stories.ts (hunk 1 above)
Comment (GiantRobots) [reply src/lib/holocene/stories/Button.stories.ts:26]: 100% They kind of were before but not sure why this wasn't typed correctly. I think it ends up being a weird generic thing

**Pull request closed without merging**